packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksEditor.tsx

```diff
@@ -27,7 +30,10 @@ import { withStrapiSchema } from './plugins/withStrapiSchema';
 
 interface NonSelectorBlock {
   renderElement: (props: RenderElementProps) => React.JSX.Element;
-  value: object;
+  value: object & {
```

Comment:
Can you explain why you've joined an object to this instead of replacing the type?

Reply:
I was not sure why @remidej typed it as an object, so just extended it. Replaced it.